test/arguments/cwd.js

```diff
@@ -70,7 +71,7 @@ if (!isWindows) {
 }
 
 const cwdNotExisting = {cwd: 'does_not_exist', expectedCode: 'ENOENT', expectedMessage: 'The "cwd" option is invalid'};
-const cwdTooLong = {cwd: '.'.repeat(1e5), expectedCode: 'ENAMETOOLONG', expectedMessage: 'The "cwd" option is invalid'};
+const cwdTooLong = {cwd: '.'.repeat(1e5), expectedCode: isWindows && majorNodeVersion >= 20 ? 'ENOENT' : 'ENAMETOOLONG', expectedMessage: 'The "cwd" option is invalid'};
```

Comment:
When calling `child_process.spawn()` with a `cwd` that has a long name, the `error.code` is `ENAMETOOLONG` on Unix. It used to be that on Windows too, but since Node 24.2.0  it's `ENOENT`. I could not find the commit in the Node changelog that changed that behavior, but it's nothing that's impacted Execa's code, so it's ok.